astropy/coordinates/tests/test_celestial_transformations.py

```diff
@@ -188,7 +188,7 @@ class TestHCRS():
     coord.ICRS(coord.get_body_barycentric(tarr, 'sun')).
     """
 
-    def setup(self):
+    def setup_method(self):
```

Comment:
Why not `setup_class`? Does this need to be redefined before every method test?

Same question for all other similar changes.

Reply:
According to https://docs.pytest.org/en/7.2.x/deprecations.html#nose-deprecation `setup_method` `teardown_method` are the replacements for `setup` and `teardown`.